docs/stdlib/set.rst

```diff
@@ -407,8 +407,8 @@ Sets
 
         select User.<owner[is Issue];
 
-        # With the use of type intersection it's possible to refer to
-        # specific property of Issue now:
+        # With the use of type intersection it's possible to refer
+        # to a specific property of Issue now:
```

Comment:
I added the missing “a” but also changed the word wrap location so that the comment would be fully visible without needing to horizontally scroll the code example when viewing the page in a medium-width browser.